ee/app/livechat-enterprise/client/views/app/customTemplates/businessHoursCustomFieldsForm.html

```diff
@@ -0,0 +1,58 @@
+<template name="businessHoursCustomFieldsForm">
```

Comment:
Not related to this template but I'd like to mention that we have the `livechatDepartmentCustomFieldsForm` and it would be very nice if we display the name of the Business Hour` used by the department also.